sample-functions/ApiKeyProtected/README.md

```diff
@@ -1,6 +1,6 @@
 ### Api-Key-Protected sample
 
-To use this sample provide an env variable for the container/service in `secret_api_key`.
+To use this sample provide a secret for the container/service in `secret_api_key` using [Docker Swarm Secret](https://docs.docker.com/engine/swarm/secrets/#defining-and-using-secrets-in-compose-files).
```

Comment:
Might read better with "in the `secret_api_key` [Docker Swarm Secret].."

I don't quite follow the docker-compose link here, is this something to cover in the guide/needed?

Reply:
I will change it to link to the guide instead.